### ISSUE TITLE
kd: SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by usirin** [koding/kd-atom#9]
1. Try to list `koding` teams using `kd:teams` command.

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.2
**Thrown From**: [kd](https://github.com/koding/kd-atom) package 0.1.12

### Stack Trace

kd: SyntaxError: Unexpected end of JSON input

```
At undefined

SyntaxError: Unexpected end of JSON input
  at Object.parse (native)
  at then.result (/packages/kd/lib/utils/get-teams.js:10:24)
  at process._tickCallback (internal/process/next_tick.js:103:7)

```

### Commands

```
     -0:13 command-palette:toggle (input.hidden-input)
     -0:10.5.0 core:confirm (input.hidden-input)
     -0:10.5.0 kd:teams (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
ariake-dark-syntax 0.1.3 
atom-alignment 0.12.1 
atom-clock 0.1.6 
atom-koding 0.0.0 
atom-pair 2.0.10 
atom-tabs-plus 0.0.0 
atom-vim-like-tab 1.3.1 
autocomplete-modules 1.6.8 
autocomplete-paths 1.0.5 
blessed-dark-syntax 0.3.0 
blessed-light-syntax 0.3.0 
busy-signal 1.3.0 
chester-atom-syntax 0.1.1 
coffee-compile 0.25.0 
formatter 2.12.4 
git-plus 7.3.3 
goto-definition 1.3.1 
haskell-grammar 0.4.0 
intentions 1.1.2 
kd 0.1.12 
keystroke 0.1.3 
language-babel 2.57.2 
language-cjsx 0.3.0 
language-svg 0.9.2 
linter 2.1.2 
linter-ui-default 1.2.2 
merge-conflicts 1.4.4 
narrow 0.39.0 
nova-atom-syntax 1.4.1 
nova-atom-ui 1.3.0 
nuclide 0.214.0 
nuclide-format-js 0.0.36 
open-this 0.4.0 
pigments 0.39.1 
platformio-ide-terminal 2.5.0 
preview 0.17.0 
quick-highlight 0.8.1 
sort-lines 0.14.0 
spacegray-light-neue-ui 0.1.7 
Stylus 3.1.0 
svg-preview 0.11.0 
sync-settings 0.8.1 
vim-mode-plus 0.86.3 
vim-mode-plus-ex-mode 0.9.0 
vim-mode-plus-keymaps-for-surround 0.2.1 
vim-mode-plus-move-to-symbols 0.2.0 
yeti-ui 0.1.6 
```

**Comment by usirin** [comment]
This is happening when there is a problem in `kd ls` command. (no running daemon, no connection, etc. etc).

**Comment by usirin** [comment]
Closed with #18
